languages/translations/uk.json

```diff
@@ -0,0 +1,421 @@
+                "name": "Приєднайтеся до потоку ліворуч по {way_name}",
+                "destination": "Приєднайтеся до потоку ліворуч у напрямку {destination}"
+            },
+            "sharp right": {
+                "default": "Приєднайтеся до потоку праворуч",
+                "name": "Приєднайтеся до потоку праворуч по {way_name}",
+                "destination": "Приєднайтеся до потоку праворуч у напрямку {destination}"
+            },
+            "uturn": {
+                "default": "Здійсніть розворот",
+                "name": "Здійсніть розворот на {way_name}",
+                "destination": "Здійсніть розворот у напрямку {destination}"
+            }
+        },
+        "new name": {
+            "default": {
+                "default": "Рухайтесь {modifier}",
+                "name": "Рухайтесь {modifier} на {way_name}",
+                "destination": "Рухайтесь {modifier} у напрямку {destination}"
+            },
+            "straight": {
+                "default": "Продовжуйте рух прямо",
+                "name": "Продовжуйте рух по {way_name}",
+                "destination": "Продовжуйте рух у напрямку {destination}"
+            },
+            "sharp left": {
+                "default": "Прийміть різко ліворуч",
+                "name": "Прийміть різко ліворуч на {way_name}",
+                "destination": "Прийміть різко ліворуч у напрямку {destination}"
+            },
+            "sharp right": {
+                "default": "Прийміть різко праворуч",
+                "name": "Прийміть різко праворуч на {way_name}",
+                "destination": "Прийміть різко праворуч у напрямку {destination}"
+            },
+            "slight left": {
+                "default": "Прийміть плавно ліворуч",
+                "name": "Рухайтесь плавно ліворуч на {way_name}",
+                "destination": "Рухайтесь плавно ліворуч у напрямку {destination}"
+            },
+            "slight right": {
+                "default": "Прийміть плавно праворуч",
+                "name": "Рухайтесь плавно праворуч на {way_name}",
+                "destination": "Рухайтесь плавно праворуч у напрямку {destination}"
+            },
+            "uturn": {
+                "default": "Здійсніть розворот",
+                "name": "Здійсніть розворот на {way_name}",
+                "destination": "Здійсніть розворот у напрямку {destination}"
+            }
+        },
+        "notification": {
+            "default": {
+                "default": "Рухайтесь {modifier}",
+                "name": "Рухайтесь {modifier} на {way_name}",
+                "destination": "Рухайтесь {modifier} у напрямку {destination}"
+            },
+            "uturn": {
+                "default": "Здійсніть розворот",
+                "name": "Здійсніть розворот на {way_name}",
+                "destination": "Здійсніть розворот у напрямку {destination}"
+            }
+        },
+        "off ramp": {
```

Comment:
If any changes are needed, please make them [in Transifex](https://www.transifex.com/project-osrm/osrm-text-instructions/language/uk/), and I’ll update this PR. Otherwise, if we make the changes directly in GitHub, they’ll get overridden the next time someone pulls translations from Transifex.